--- OpenMoteFirmware/tools/openmote-bsl/openmote-bsl.py	2016-01-25 15:51:30.363310728 +0100
+++ OpenMoteFirmware/tools/openmote-bsl/openmote-bsl.py	2016-01-25 15:51:30.363310728 +0100
@@ -25,6 +25,8 @@
 
 # Define the relative path to the project home
 openmote_bsl_path = "../../tools/openmote-bsl"
+if not os.path.isdir(openmote_bsl_path):
+    openmote_bsl_path = "../OpenMoteFirmware/tools/openmote-bsl"
 
 # Define logging file name, path and configuration
 log_name = "openmote-bsl.cfg"
